prepareShapesAndCards.py

```diff
@@ -529,20 +545,51 @@ def prepareShapes(backgrounds, signals, discriminant, discriminantName):
                 else:
                     #cb.cp().AddSyst(cb, '$PROCESS_'+systematic, 'shape', ch.SystMap('process')(['ttother', 'ttlf', 'ttbj', 'tthad', 'ttfullLep'], 1.00))
                     cb.cp().AddSyst(cb, systematic, 'shape', ch.SystMap('process')(smTTlist, 1.00))
-            cb.cp().AddSyst(cb, '$ERA_lumi', 'lnN', ch.SystMap('era')(['%s'%options.dataYear], options.luminosityError))
+#            cb.cp().AddSyst(cb, '$ERA_lumi', 'lnN', ch.SystMap('era')(['%s'%options.dataYear], options.luminosityError))
+            cb.cp().AddSyst(cb, 'CMS$ERA_lumi', 'lnN', ch.SystMap()(options.luminosityError))
             cb.cp().AddSyst(cb, 'tt_xsec', 'lnN', ch.SystMap('process')
                     (['ttbb', 'ttcc', 'ttlf'], 1.055)
                     )
-            cb.cp().AddSyst(cb, '$PROCESS_norm', 'lnN', ch.SystMap('process')
-                    (['ttbb'], 1.3)
-                    )
-            cb.cp().AddSyst(cb, '$PROCESS_norm', 'lnN', ch.SystMap('process')
-                    (['ttcc'], 1.5)
-                    )
+#            cb.cp().AddSyst(cb, '$PROCESS_norm', 'lnN', ch.SystMap('process')
+#                    (['ttbb'], 1.3)
+#                    )
+#            cb.cp().AddSyst(cb, '$PROCESS_norm', 'lnN', ch.SystMap('process')
+#                    (['ttcc'], 1.5)
+#                    )
             cb.cp().AddSyst(cb, 'Other_xsec', 'lnN', ch.SystMap('process')
                     #(['SingleTop', 'ttV', 'Wjets', 'DYjets', 'VV', 'tth'], 1.1)
                     (['other'], 1.1)
                     )
+            if options.dataYear == '2016':
+                cb.cp().AddSyst(cb, 'hdamp_2016', 'lnN', ch.SystMap('process')
+                        (['ttbb', 'ttcc', 'ttlf'], 1.05)
+                        )
+                cb.cp().AddSyst(cb, 'scale_2016', 'lnN', ch.SystMap('process')
+                        (['ttbb', 'ttcc', 'ttlf'], 1.15)
+                        )
+                for i in xrange(len(discriminant)):
+                    cb.cp().AddSyst(cb, '$PROCESS_norm_'+str(discriminant[i][1]), 'lnN', ch.SystMap('process')
```

Comment:
Does'nt this leads to norm uncertainty uncorrelated for all jet bins while it was one for j3* and one for j4*?

Reply:
Arr ... this is my mistake. I will correct it